x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -262,21 +262,23 @@ export function ProcessTreeNode({
               <EuiIcon type="user" /> <b css={styles.darkText}>{dataOrDash(user?.name)}</b>
             </>
           ) : (
-            <span>
+            <>
               {showTimestamp && (
                 <span data-test-subj="sessionView:processTreeNodeTimestamp" css={styles.timeStamp}>
                   {timeStampsNormal}
                 </span>
               )}
               <EuiToolTip position="top" content={iconTooltip}>
                 <EuiIcon data-test-subj={iconTestSubj} type={processIcon} css={styles.icon} />
-              </EuiToolTip>{' '}
-              <span ref={textRef}>
-                <span css={styles.workingDir}>{dataOrDash(workingDirectory)}</span>&nbsp;
-                <span css={styles.darkText}>{dataOrDash(args?.[0])}</span>{' '}
-                {args?.slice(1).join(' ')}
+              </EuiToolTip>
+              <span ref={textRef} css={styles.textSection}>
+                <span css={styles.workingDir}>{dataOrDash(workingDirectory)}</span>
+                <span css={styles.textSpacing}>&nbsp;</span>
```

Comment:
Wondering if it would be better to include a `marginRight` for workingDir style above instead of having an element just for spacing

Reply:
yea seems a little strange to set the width of a span with a nbsp inside. Is the width of a regular space not aligned with the design?

Reply:
@zizhouW didn't use `marginRight`, because the margin would cut off the spacing when the user attempt to copy it.

For example, when copying this text:

```
/home/vagrant ls --color=auto
```

it turn into:
```
/home/vagrantls--color=auto
```

Reply:
@opauloh We added custom padding to make it easier to scan information, distinguish the working directory, code, and properties in one line. In design `working directory` `command` `property` are separate elements with `6px `paddings between them. The same approach we had in Cmd. In addition, this padding is a component, so we can change the width flexibly, regardless of the font.

I see in this discussion that it causes certain difficulties. If the logic is broken and it affects user experience (ex. copying the command), we can consider the option with natural font space.

Reply:
thanks for the explanation @codearos! this PR introduces a special component for spacing, and as you said, the benefit is that it can be adjusted to fit the needs without hurting UX, so we are keeping with it